docs/implementation-guide.md

```diff
@@ -57,65 +57,47 @@ In this section we describe how a STLT can take this repository and use it to sp
 
-Optionally, you may also install the GitHub CLI tool (`gh`) to enable automatically setting secrets in your repository. Follow [this guide](https://cli.github.com/manual/installation) to install the tool. Confirm that the installation was successful by running `gh --version`. If `gh` is installed properly you should get a response similar to what is shown below.
+TEMP/PLACEHOLDER vars - Need confirmation on these and any others required
 
-```bash
-❯ gh --version
-gh version 2.14.7 (2022-08-25)
-https://github.com/cli/cli/releases/tag/v2.14.7
-```
 
-### Step 3: Fork the phdi-google-cloud Repository
-Fork the phdi-google-cloud repository into your organization's, or your personal, GitHub account.
-1. Navigate to [https://github.com/CDCgov/phdi-google-cloud](https://github.com/CDCgov/phdi-google-cloud).
-2. Click on the `Fork` button in the top right.
-![fork-repo-1](./images/fork-repo-1.png)
-3. Select the owner of the forked repository. We recommend that this is an organization and not an individual.
-4. Click `Create fork` at the bottom of the page.
-![fork-repo-2](./images/fork-repo-2.png)
+It is not recommended to exit the quick start script partway through so please have all values on hand when you run the script.
+Each of the following will require a value during the process:
+- `PROJECT_ID` - Specified by the quick start script.
+- `SERVICE_ACCOUNT_ID` - Specified by the quick start script.
+- `WORKLOAD_IDENTITY_PROVIDER` - Specified by the quick start script.
+- `REGION` - Your choice of GCP region; quick start script suggests `us-central1`.
+- `ZONE`- Your choice of GCP zone; quick start script suggests `us-central1-a`.
+
+(Optional) Required to use geocoding functionality:
+- `SMARTY_AUTH_ID` - Your SmartyStreet Authorization ID. More info on the Smarty geocoding service [here](https://www.smarty.com/pricing/us-rooftop-geocoding)
+- `SMARTY_AUTH_TOKEN` - Your SmartyStreet Authorization Token.
 
-### Step 4: Clone the Forked Repository
-Clone the forked version of the phdi-google-cloud repository by running `git clone https://github.com/<MY-GITHUB-ORGANIZATION>/phdi-google-cloud.git`.
 
-### Step 5: Run the Quick Start Script
+### Step 2: Run the Quick Start Script in Google Cloud Terminal
 In this step we will work through GCP's [Workload Identity Federation](https://cloud.google.com/iam/docs/workload-identity-federation) to grant your phdi-google-cloud repo access to deploy the pipelines to your organization's GCP environment. We have provided a script to automate most of this process that we recommend you use. However, if you prefer to work through it manually you may follow [this guide](https://github.com/google-github-actions/auth#setup).
 
-From your machine's command line:
+Navigate to the Google Cloud Dashboard and terminal
+
+GOOGLE CLOUD LINK, MAYBE WITH HIGHLIGHTED SCREENS SHOWING TERMINAL LOCATION AND MORE EDITS HERE. Is repo cloning fully handled by the script atm?
```

Comment:
https://console.cloud.google.com/
<img width="1575" alt="Screenshot 2022-11-28 at 5 32 38 PM" src="https://user-images.githubusercontent.com/9121162/204416448-0891765f-18f3-40bc-950b-f871dc716065.png">

The user will need to initially clone _our_ repo to get the quickstart script into the Cloud Shell, but then the script will handle forking.